gradle/libs.versions.toml

```diff
@@ -8,7 +8,7 @@ androidxCore = "1.7.0"
 androidxFragment = "1.5.0"
 androidxLifecycle = "2.5.0"
 androidxWork = "2.7.0"
-anvil = "2.5.0-beta01"
+anvil = "2.5.0-beta03"
```

Comment:
fyi: [v2.5.0-beta04](https://github.com/square/anvil/releases/tag/v2.5.0-beta04) is out already. 😏